conf/v1.routes

```diff
@@ -12,4 +12,7 @@ DELETE     /foreign/:nino/:taxYear                          v1.controllers.Delet
 
 GET        /pensions/:nino/:taxYear                         v1.controllers.RetrievePensionsReliefsController.handleRequest(nino: String, taxYear: String)
 PUT        /pensions/:nino/:taxYear                         v1.controllers.AmendPensionsReliefsController.handleRequest(nino: String, taxYear: String)
-DELETE     /pensions/:nino/:taxYear                         v1.controllers.DeletePensionsReliefsController.handleRequest(nino: String, taxYear: String)
\ No newline at end of file
+DELETE     /pensions/:nino/:taxYear                         v1.controllers.DeletePensionsReliefsController.handleRequest(nino: String, taxYear: String)
+
+# see v1WithCharitableGiving.routes:
+# PUT        /charitable-giving/:nino/:taxYear                v1.controllers.CreateAndAmendCharitableGivingController.handleRequest(nino: String, taxYear: String)
```

Comment:
Probably best to remove the commented code from the routes until we need to put them in